src/api/entityResolution/index.ts

```diff
@@ -148,6 +150,24 @@ export class EntityResolutionAPI extends Request {
     });
   }
 
+  /**
+   * Stop an ingestion task on a given data-source.
+   */
+  stopIngestion(this: Request<IngestionStatus>, params: StopEntityResolutionTaskParams) {
+    return this.request({
+      errors: [
+        UNAUTHORIZED,
+        FORBIDDEN,
+        DATA_SOURCE_UNAVAILABLE,
```

Comment:
if the source is offline, wouldn’t the ER be stopped by itself?

Reply:
yes, but also the API can be called with a sourceKey of an offline datasource